### PR TITLE
V1.0.3 backup VLAN trunk implementation

### PR DESCRIPTION
**Pull request by JipanYanga**

### CODE REVIEW

**Comment by lguohan** [comment]
@taoyl-ms  to review

**Comment by stcheng** [comment]
Hi Jipan, is it possible that you could split this pull request into multiple related pull requests, e.g. one for the orchagent change, one for the config agent, etc.

**Comment by JipanYanga** [comment]
Hi Shuotian,

Technically I could split the changes into two parts, one for orchagent and the other for config agent.    The hostenv setup in congfigAgent and ASIC setup from Orchagent need to work together to get VLAN trunk functioning.  

What is the purpose of this splitting?  If it is for simplifying the review, I could do it.  The commit history log for VLAN trunk will be lost.

Thanks,
Jipan

**Comment by stcheng** [comment]
I think this is a large feature. here you squash them into one commit. It is already losing the commit history. Either splitting the pull request, or have a better commit history so we do a regular merge commit instead of a squash one. This will benefit both git commit view and pull request review.

**Comment by stcheng** [comment]
Hello, @JipanYanga , sorry for the confusion. I'm referring to the sonic-swss code changes.

**Comment by taoyl-ms** [comment]
https://github.com/Azure/sonic-swss/pull/294

File: src/libnl3/Makefile [hunk 1]
```diff
@@ -4,6 +4,7 @@ SHELL = /bin/bash
 
 MAIN_TARGET = libnl-3-200_$(LIBNL3_VERSION)_amd64.deb
 DERIVED_TARGETS = libnl-3-dev_$(LIBNL3_VERSION)_amd64.deb \
+		  libnl-3-200-dbg_$(LIBNL3_VERSION)_amd64.deb \
```
Comment (stcheng) [inline src/libnl3/Makefile]: why adding dbg here?

Comment (JipanYanga) [reply src/libnl3/Makefile]: Have been seeing issues related to netlink, this to get the dbg file copied to target/debs folder. It makes it easier to find the libnl debug package and install it when needed.

**Comment by JipanYanga** [comment]
target branch not active.

**Pull request closed without merging**